BMS_checker.cpp

```diff
@@ -30,13 +32,15 @@ void BMS :: Verify_Parameter(float parameter,BMS_Parameters name_en)
   const char* Parameter_Name = BMS_Parameters_Names[BMS_Market_Language_en][name_en];
   if(parameter < min)
   {
-    cout <<"Low "<<Parameter_Name<<" Breach! \n";
-    BMS_OK = false;
+    //cout <<"Low "<<Parameter_Name<<" Breach! \n";
```

Comment:
Do not leave commented code in commits.
If you need it, you can find it in the history of source-control.